### ISSUE TITLE
2.6.0 breaks with missing files

### ISSUE THREAD

**Issue by png262** [bee-queue/arena#154]
just upgraded to arena 2.6.0, and my app breaks:
Error: ENOENT: no such file or directory, open '/Users/xxxxx/Projects/xxxxxxxx/node_modules/bull-arena/public/vendor/tablesort.min.js'

reverted back to 2.5.4 and it works again.

**Comment by engvik** [comment]
I'm not able to install `2.6.0`:

Node: `v10.15.1`
NPM: `6.4.1`

```
npm ERR! path /home/xxxxx/node_modules/highlight.js/tools/build.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/xxxxx/node_modules/highlight.js/tools/build.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

`2.5.4` works fine for me as well.

**Comment by Dids** [comment]
```
Error: ENOENT: no such file or directory, open '/<project_path>/node_modules/bull-arena/public/vendor/tablesort.min.js'
```

**Comment by iredjee** [comment]
`public/vendor` directory is missing from NPM registry

**Comment by lehno** [comment]
My build is broken, need to comment bull arena for now to get things working. Any update on this? Also I'm very interested in contribute with this project. Will send some pull requests soon

**Comment by rpavez** [comment]
Me and @pbassut having same issue

**Comment by hujung** [comment]
https://github.com/tristen/tablesort you can copy tablesort.min.js under public/vendor and copy img folder too from the master branch.

**Comment by ttacon** [comment]
Apologies for this! I've released v2.6.1 to fix this (see #155).
